spec/mailers/previews/mentee_application_mailer_preview.rb

```diff
@@ -1,24 +1,34 @@
 # Preview all emails at http://localhost:3000/rails/mailers/mentee_application_mailer
 class MenteeApplicationMailerPreview < ActionMailer::Preview
-  def notify_for_acceptance
+  def notify_applicant_of_submission
     application = UserMenteeApplication.last
-    MenteeApplicationMailer.with(recipient: application.user, application:).notify_for_acceptance
+    MenteeApplicationMailer.with(recipient: application.user, application:).notify_applicant_of_submission
   end
 
-  def notify_for_rejection
-    MenteeApplicationMailer.with(recipient: User.last).notify_for_rejection
+  def notify_admin_of_submission
+    application = UserMenteeApplication.last
+    MenteeApplicationMailer.with(recipient: User.admin.last, application:).notify_admin_of_submission
+  end
+
+  def notify_applicant_of_acceptance
+    application = UserMenteeApplication.last
+    MenteeApplicationMailer.with(recipient: application.user, application:).notify_applicant_of_acceptance
+  end
+
+  def notify_applicant_of_rejection
+    MenteeApplicationMailer.with(recipient: User.last).notify_applicant_of_rejection
   end
 
-  def notify_for_code_challenge_sent
-    MenteeApplicationMailer.with(recipient: User.last).notify_for_code_challenge_sent
+  def notify_applicant_of_code_challenge_sent
+    MenteeApplicationMailer.with(recipient: User.last).notify_applicant_of_code_challenge_sent
   end
 
-  def notify_for_code_challenge_approved
-    MenteeApplicationMailer.with(recipient: User.last).notify_for_code_challenge_approved
+  def notify_applicant_of_code_challenge_approved
+    MenteeApplicationMailer.with(recipient: User.last).notify_applicant_of_code_challenge_approved
   end
 
-  def notify_for_reapplication
+  def notify_applicant_to_reapply
```

Comment:
this naming is a modification of the proposed scheme. I suppose it could be renamed to be something like `notify_applicant_of_new_cohort`, if we really want to be sticklers for this naming scheme.